src/Vector3_TEST.cc

```diff
@@ -15,6 +15,9 @@
  *
 */
 
+#include <numeric>
+#include <sstream>
+
 #include <gtest/gtest.h>
```

Comment:
The code checker is complaining here:

```
  ./src/Vector3_TEST.cc:21:  Found C system header after C++ system header. Should be: Vector3_TEST.h, c system, c++ system, other.  [build/include_order] [4]
```

Reply:
Sorry, I neglected to do the code check.